src/Security/Authentication/JwtBearer/src/LoggingExtensions.cs

```diff
@@ -13,4 +13,7 @@ internal static partial class LoggingExtensions
 
     [LoggerMessage(3, LogLevel.Error, "Exception occurred while processing message.", EventName = "ProcessingMessageFailed")]
     public static partial void ErrorProcessingMessage(this ILogger logger, Exception ex);
+
+    [LoggerMessage(4, LogLevel.Information, "Forbidden response has started.", EventName = "ForbiddenResponseHasStarted")]
```

Comment:
```suggestion
    [LoggerMessage(4, LogLevel.Debug, "Unable to reject the response as forbidden, it has already started.", EventName = "ForbiddenResponseHasStarted")]
```

Reply:
Thank you for the feedback @Tratcher .  I implemented this suggested change when I pushed up the modification to the unit test.